chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -20,22 +20,6 @@
             <name>downloadFileJob2011</name>
             <group>systemJobs</group>
         </delete-job>
-        <delete-job>
-            <name>g3Sed2015DownloadFileJob</name>
-            <group>systemJobs</group>
-        </delete-job>
-        <delete-job>
-            <name>surveillanceDownloadFileJob</name>
-            <group>systemJobs</group>
-        </delete-job>
-        <delete-job>
-            <name>directReviewDownloadFileGeneration</name>
-            <group>systemJobs</group>
-        </delete-job>
-        <delete-job>
-            <name>removeCriteriaJob</name>
-            <group>systemJobs</group>
-        </delete-job>
         <!-- END remove after PROD push -->
```

Comment:
Do we need to add a "delete" command to remove the "downloadFileJob2015" old job?

Reply:
It's already there. It doesn't show as a diff because it was there from some previous ticket and never removed, so I left it.